uiplib/Wallpaper.py

```diff
@@ -52,10 +52,10 @@ def set_wallpaper_osx(self, filename):  # pragma: no cover
     def get(self):
         """Get file address of current Wallpaper."""
         if sys.platform.startswith('win32'):   # pragma: no cover
-            return self.get_wallpaper_windows()
+            return [self.get_wallpaper_windows()]
 
```

Comment:
Why do we need a list here?

Reply:
to keep it consistent with others, your function should only return one type of value ever, so the calling code operates consistently.

Reply:
Plus there is a huge probability we might have get return a list of different desktop wallpapers i dont want it to change the contract.

Reply:
But, you've excluded `OS X` from all of this. :P